crypto/x509/x509_vfy.c

```diff
@@ -314,9 +314,10 @@ static int sk_X509_contains(STACK_OF(X509) *sk, X509 *cert)
 }
 
 /*
- * Find in given STACK_OF(X509) sk a non-expired issuer cert (if any) of given cert x.
+ * Find in given STACK_OF(X509) sk an issuer cert (if any) of given cert x.
```

Comment:
Suggestion : using Javadoc style can improve readability here : 
`Find in given STACK_OF(X509)* |sk| an issuer cert (if any) of given cert |x|.`

Reply:
Thank you for this good hint.
I like this style improvement - done.